### ISSUE TITLE
Recomendasions for OS and Setup 

### ISSUE THREAD

**Issue by Habnone** [ef3d0c3e/SvenHax#12]
What Linux Distro I should use that does not give me autisim when trying to compile and run it, just what distro it 100% works on

**Comment by ef3d0c3e** [comment]
Note that this is an incomplete project and currently only AA and BHop  works.

> Works 100%

Debian sid (the one I use).
It is difficult to set up correctly and packages have many bugs which you have to watch for on a bug list
Plus it's a pain to keep updated, and depending on your hardware might be even more painful.
> Does not give autism

You should be able to use any distro that provides gcc >= 10.2 and gdb>=10.1.
So anything that's been updated in the last 1.5years (so no distro with the update cycle of debian  stable, like Mint).
However,  debian stable should be getting updated packages from sid/testing in june 2023 for the bookworm release.

tldr: use a rolling release distro.

**Comment by Habnone** [comment]
Alr so if only AA and bhop works then ig its not worth setting up, thanks for the info tho, understandable why you arent keeping it updated
